src/cli/commands/force/lightning/lwc/previewConfigurationSchema.json

```diff
@@ -0,0 +1,102 @@
+{
+  "$id": "https://developer.salesforce.com/mobile-tooling/preview-configuration.schema.json",
```

Comment:
We should probably version our schema

Reply:
That seems like a fine idea. How about `https://developer.salesforce.com/mobile-tooling/1.0/preview-configuration.schema.json`?